kaniko.go

```diff
@@ -224,7 +260,137 @@ func (p Plugin) Exec() error {
 	if p.Build.TarPath != "" {
 		cmdArgs = append(cmdArgs, fmt.Sprintf("--tar-path=%s", p.Build.TarPath))
 	}
-	
+
+	if p.Build.CacheCopyLayers {
+		cmdArgs = append(cmdArgs, "--cache-copy-layers")
+	}
+
+	if p.Build.CacheRunLayers {
+		cmdArgs = append(cmdArgs, "--cache-run-layers=true")
+	}
+
+	if p.Build.Cleanup {
+		cmdArgs = append(cmdArgs, "--cleanup=true")
+	}
+
+	_, ok := os.LookupEnv("PLUGIN_COMPRESSED_CACHING")
```

Comment:
why are we looking up the env variable?

Reply:
this is because, we want to set the flag to true or false only if its set,

Reply:
Wouldn't `flag.IsSet` return `false` if the flag does not come from the env? And then we could set the pointer accordingly?

Reply:
I see, I have changed and added this logic, working as expected with true and false both.